### PR TITLE
Replace multiple boolean arguments to one uint8_t.

### PR DESCRIPTION
**Pull request by zherczeg**

### CODE REVIEW

**Comment by zherczeg** [comment]
Binary size decreased by 1K.

File: jerry-core/ecma/base/ecma-helpers.c [hunk 1]
```diff
@@ -576,26 +576,13 @@ ecma_get_internal_property (ecma_object_t *object_p, /**< object descriptor */
 ecma_property_t *
 ecma_create_named_data_property (ecma_object_t *object_p, /**< object */
                                  ecma_string_t *name_p, /**< property name */
-                                 bool is_writable, /**< 'Writable' attribute */
-                                 bool is_enumerable, /**< 'Enumerable' attribute */
-                                 bool is_configurable) /**< 'Configurable' attribute */
+                                 uint8_t prop_attributes) /**< property attributes */
```
Comment (LaszloLango) [inline jerry-core/ecma/base/ecma-helpers.c]: I like this idea, but I'd give a name to this. What do you think about `typedef uint8_t prop_attr_t;` or `typedef uint8_t prop_attr_flag_t;`?

Comment (LaszloLango) [inline jerry-core/ecma/base/ecma-helpers.c]: I just saw that we already have a `ecma_property_flags_t`, but it is an enum, so it'll be 32bit long (usually, but not always). In this case I think it is unnecessary to create a new type, but it should be mentioned in the comment.

Comment (zherczeg) [inline jerry-core/ecma/base/ecma-helpers.c]: That is a flagset, not a flag. The type_and_flags in ecma_property_t is also an uint8_t. I wouldn't want to change it.

**Comment by zherczeg** [comment]
Any more thoughts?

File: jerry-core/ecma/base/ecma-helpers.c (hunk 1 above)
Comment (LaszloLango) [inline jerry-core/ecma/base/ecma-helpers.c]: As I said, you don't have to change the name, just make a comment.

```c
...
uint8_t prop_attributes) /**< property attributes (See: ecma_property_flags_t) */
```

**Comment by LaszloLango** [comment]
LGTM, after the comment fixed.

**Comment by dbatyai** [comment]
LGTM

**Pull request merged**